src/app/Models/Profile.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\HasOne;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Profile extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user_id',
+        'first_name',
+        'last_name',
+        'birth_date',
+        'address',
+        'contact'
+    ];
+
+    public function user(): HasOne
```

Comment:
1.) Add docbloc without param and returns

Reply:
Done